### ISSUE TITLE
[BUG] Crash caused by AutoTotem

### ISSUE THREAD

**Issue by mumy-255** [CCBlueX/LiquidBounce#2713]
### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.2.0 git-e46d279

### Operating System

Windows 10

### Minecraft Version

1.20.4

### Describe the bug

When you activate AutoTotem in creative mode, opening the inventory and taking out a totem while your off-hand is not a totem will cause the game to stop.

### Steps to reproduce

...

### Client Log

```shell
[09:46:13] [Server thread/WARN]: Can't keep up! Is the server overloaded? Running 2085ms or 41 ticks behind
[09:46:14] [Render thread/INFO]: Reloading entity icon resources...
[09:46:14] [Render thread/INFO]: Done!
[09:46:14] [Render thread/WARN]: Time from main menu to in-game was 5.781474 seconds
[09:46:14] [Render thread/WARN]: Total time to load game and open world was 77.51847 seconds
[09:46:14] [Render thread/INFO]: Loaded 327 advancements
[09:46:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ: [id: 0x6b060fab, L:/127.0.0.1:16226 - R:/127.0.0.1:5396]
[09:46:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ COMPLETE
[09:46:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ: [id: 0xe1cdbb40, L:/127.0.0.1:16226 - R:/127.0.0.1:5398]
[09:46:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ COMPLETE
[09:46:18] [Server thread/INFO]: [mumy_: 已将自己的游戏模式设置为创造模式]
[09:46:18] [Render thread/INFO]: [CHAT] 已将自己的游戏模式设置为创造模式
[09:46:23] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ: [id: 0x246f09f4, L:/127.0.0.1:16226 - R:/127.0.0.1:5403]
[09:46:23] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ COMPLETE
[09:46:24] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-2/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-2/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-2/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:24] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:25] [nioEventLoopGroup-4-2/INFO]: Successfully saved config 'modules'.
[09:46:25] [nioEventLoopGroup-4-2/INFO]: Successfully saved config 'modules'.
[09:46:25] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:46:35] [Render thread/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-2/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:37] [nioEventLoopGroup-4-5/INFO]: Successfully saved config 'modules'.
[09:46:45] [Server thread/INFO]: Saving and pausing game...
[09:46:45] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[09:46:45] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[09:46:45] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[09:47:11] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:47:11] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:47:11] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:47:11] [nioEventLoopGroup-4-4/INFO]: Successfully saved config 'modules'.
[09:47:11] [nioEventLoopGroup-4-4/INFO]: Successfully saved config 'modules'.
[09:47:11] [nioEventLoopGroup-4-4/INFO]: Successfully saved config 'modules'.
[09:47:11] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:47:11] [nioEventLoopGroup-4-3/INFO]: Successfully saved config 'modules'.
[09:47:12] [Render thread/INFO]: Successfully saved config 'modules'.
[09:47:14] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ: [id: 0x83c65d05, L:/127.0.0.1:16226 - R:/127.0.0.1:5458]
[09:47:14] [nioEventLoopGroup-3-1/INFO]: [id: 0xb59470cf, L:/[0:0:0:0:0:0:0:0]:16226] READ COMPLETE
[09:47:19] [Server thread/WARN]: Can't keep up! Is the server overloaded? Running 3003ms or 60 ticks behind
```

### Screenshots

https://github.com/CCBlueX/LiquidBounce/assets/161578211/681cc1b5-0d97-4600-bb23-3a51ffb636fc

**Comment by superblaubeere27** [comment]
I cannot replicate this issue

**Comment by mumy-255** [comment]
> I cannot replicate this issue

This is the mods I am using.

```shell
[04:27:08] [main/INFO]: Loading 181 mods:
	- appleskin 2.5.1+mc1.20.3
	- badoptimizations 2.1.0
	- betterbeds 1.3.0
	- bettermounthud 1.2.2
	- betterpingdisplay 1.1.1
	- betterstats 3.9.5+fabric-1.20.4
	   \-- tcdcommons 3.9.4+fabric-1.20.4
	- bookshelf 23.0.6
	- bugjump 1.2.0-1.20.1
	- carpet 1.4.128+v231205
	- carpet-extra 1.4.128
	- carpet-fixes 1.20-1.17.0
	   |-- conditional-mixin 0.3.1
	   \-- mixinextras 0.3.5
	- cloth-config 13.0.121
	   \-- cloth-basic-math 0.6.1
	- continuity 3.0.0-beta.4+1.20.2
	- cull-less-leaves 1.3.0
	- cupboard 1.20.1-2.6
	- customskinloader 14.19.1
	- debugify 1.20.4+1.1
	- disable_custom_worlds_advice 5.0
	- experimentalperformance 0.1.1
	   |-- io_github_classgraph_classgraph 4.8.162
	   \-- org_openjdk_jol_jol-core 0.17
	- fabric-api 0.96.4+1.20.4
	   |-- fabric-api-base 0.4.36+78d798af4f
	   |-- fabric-api-lookup-api-v1 1.6.49+82b1bb3e4f
	   |-- fabric-biome-api-v1 13.0.16+78d798af4f
	   |-- fabric-block-api-v1 1.0.16+3e2216cb4f
	   |-- fabric-block-view-api-v2 1.0.4+78d798af4f
	   |-- fabric-blockrenderlayer-v1 1.1.46+78d798af4f
	   |-- fabric-client-tags-api-v1 1.1.7+78d798af4f
	   |-- fabric-command-api-v1 1.2.41+f71b366f4f
	   |-- fabric-command-api-v2 2.2.20+78d798af4f
	   |-- fabric-commands-v0 0.2.58+df3654b34f
	   |-- fabric-containers-v0 0.1.86+df3654b34f
	   |-- fabric-content-registries-v0 5.0.14+78d798af4f
	   |-- fabric-convention-tags-v1 1.5.10+78d798af4f
	   |-- fabric-crash-report-info-v1 0.2.23+78d798af4f
	   |-- fabric-data-attachment-api-v1 1.1.3+b90db5744f
	   |-- fabric-data-generation-api-v1 13.2.3+5c0133444f
	   |-- fabric-dimensions-v1 2.1.61+78d798af4f
	   |-- fabric-entity-events-v1 1.6.1+09fc25014f
	   |-- fabric-events-interaction-v0 0.7.1+389931eb4f
	   |-- fabric-events-lifecycle-v0 0.2.74+df3654b34f
	   |-- fabric-game-rule-api-v1 1.0.46+78d798af4f
	   |-- fabric-item-api-v1 2.2.0+d6f2b0844f
	   |-- fabric-item-group-api-v1 4.0.25+58f8c0124f
	   |-- fabric-key-binding-api-v1 1.0.41+78d798af4f
	   |-- fabric-keybindings-v0 0.2.39+df3654b34f
	   |-- fabric-lifecycle-events-v1 2.3.0+a67ffb5d4f
	   |-- fabric-loot-api-v2 2.1.8+78d798af4f
	   |-- fabric-message-api-v1 6.0.5+78d798af4f
	   |-- fabric-mining-level-api-v1 2.1.64+78d798af4f
	   |-- fabric-model-loading-api-v1 1.0.8+78d798af4f
	   |-- fabric-models-v0 0.4.7+9386d8a74f
	   |-- fabric-networking-api-v1 3.1.7+2e5ac5484f
	   |-- fabric-object-builder-api-v1 13.0.13+080016e44f
	   |-- fabric-particles-v1 1.1.7+78d798af4f
	   |-- fabric-recipe-api-v1 2.0.20+78d798af4f
	   |-- fabric-registry-sync-v0 4.0.19+58f8c0124f
	   |-- fabric-renderer-api-v1 3.2.4+78d798af4f
	   |-- fabric-renderer-indigo 1.5.4+78d798af4f
	   |-- fabric-renderer-registries-v1 3.2.53+df3654b34f
	   |-- fabric-rendering-data-attachment-v1 0.3.42+73761d2e4f
	   |-- fabric-rendering-fluids-v1 3.1.1+e761c6694f
	   |-- fabric-rendering-v0 1.1.56+df3654b34f
	   |-- fabric-rendering-v1 3.2.0+6fd945a04f
	   |-- fabric-resource-conditions-api-v1 2.3.14+78d798af4f
	   |-- fabric-screen-api-v1 2.0.17+78d798af4f
	   |-- fabric-screen-handler-api-v1 1.3.55+78d798af4f
	   |-- fabric-sound-api-v1 1.0.17+78d798af4f
	   |-- fabric-transfer-api-v1 4.0.11+e84342304f
	   \-- fabric-transitive-access-wideners-v1 5.0.14+78d798af4f
	- fabric-language-kotlin 1.10.18+kotlin.1.9.22
	   |-- org_jetbrains_kotlin_kotlin-reflect 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.9.22
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.23.2
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.5.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.2
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.2
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.2
	- fabricloader 0.15.7
	   \-- mixinextras 0.3.5
	- farsight 1.20.2-4.1
	   \-- org_jctools_jctools-core 4.0.1
	- faster_entity_animations 1.6
	- fastipping 1.0.1
	- fastquit 3.0.0+1.20.4
	- forgeconfigapiport 20.4.3
	   |-- com_electronwill_night-config_core 3.6.6
	   \-- com_electronwill_night-config_toml 3.6.6
	- ibeeditor 2.2.8
	- icterine 1.3.0
	- immediatelyfast 1.2.11+1.20.4
	   \-- net_lenni0451_reflect 1.3.2
	- indium 1.0.30+mc1.20.4
	- java 21
	- krypton 0.2.6
	   \-- com_velocitypowered_velocity-native 3.2.0-SNAPSHOT
	- ksyxis 1.2.2
	- lambdabettergrass 1.5.2+1.20.1
	- lambdynlights 2.3.4+1.20.4
	   |-- pride 1.2.0+1.19.4
	   \-- spruceui 5.0.3+1.20.2
	- languagereload 1.5.10+1.20.3
	- lazydfu 0.1.3
	- liquidbounce 0.2.0
	- lithium 0.12.1
	- logical_zoom 0.0.22
	- malilib 0.18.1
	- memoryleakfix 1.1.5
	- minecraft 1.20.4
	- modelfix 1.14
	- modernfix 5.15.0+mc1.20.4
	- modmenu 9.0.0
	- morechathistory 1.3.0
	- nbt_ac 1.2
	- nbttooltips 0.1.4-1.20.x
	- nochatreports 1.20.4-v2.5.0
	- noxesium 1.2.0+669ccb7
	   \-- com_noxcrew_noxesium_api_ 1.2.0+669ccb7
	- packetfixer 1.2.4
	- particletweaks 1.1.3-Fabric+1.20.2-unstable
	- reeses-sodium-options 1.7.2+mc1.20.4-build.102
	- shulkerboxtooltip 4.0.8+1.20.4
	- sodium 0.5.8+mc1.20.4
	- sodium-extra 0.5.4+mc1.20.4-build.116
	   |-- caffeineconfig 1.3.0+1.17
	   \-- crowdin-translate 1.4+1.19.3
	- sspb 3.2.1
	- starlight 1.1.3+fabric.f5dcd1a
	- stendhal 1.4.1-1.20.2
	   \-- libgui 9.0.0+1.20.2
	        |-- jankson 6.0.0+j1.2.3
	        |    \-- blue_endless_jankson 1.2.3
	        \-- libninepatch 1.2.0
	- toolstats 19.0.3
	- tooltipfix 1.1.1-1.20
	- tweakeroo 0.19.1
	- viafabricplus 3.1.1-SNAPSHOT
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viabackwards-common 4.10.0-24w09a-SNAPSHOT
	   |-- com_viaversion_viaversion-common 4.10.0-24w09a-SNAPSHOT
	   |-- de_florianmichael_classic4j 2.0.2
	   |-- fabric-resource-loader-v0 0.11.20+df798a894f
	   |-- io_jsonwebtoken_jjwt-api 0.12.5
	   |-- io_jsonwebtoken_jjwt-gson 0.12.5
	   |-- io_jsonwebtoken_jjwt-impl 0.12.5
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_commons_httpclient 1.4.1
	   |-- net_lenni0451_mcping 1.4.0
	   |-- net_lenni0451_mcstructs-bedrock_forms 1.2.0
	   |-- net_lenni0451_mcstructs-bedrock_text 1.2.0
	   |-- net_lenni0451_reflect 1.3.2
	   |-- net_raphimc_minecraftauth 4.0.0-SNAPSHOT
	   |-- net_raphimc_viaaprilfools 2.0.11-SNAPSHOT
	   |-- net_raphimc_viabedrock 0.0.6-SNAPSHOT
	   |-- net_raphimc_vialegacy 2.2.22-SNAPSHOT
	   |-- net_raphimc_vialoader 2.2.13-SNAPSHOT
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR1-SNAPSHOT
	   |-- org_iq80_leveldb_leveldb 0.12
	   |-- org_iq80_leveldb_leveldb-api 0.12
	   |-- org_lz4_lz4-pure-java 1.8.0
	   \-- org_yaml_snakeyaml 2.2
	- videotape 1.2.1
	   \-- com_github_krlite_pierced v1.2
	- xaerominimap 24.0.3
	- yet_another_config_lib_v3 3.3.2+1.20.4
	   |-- com_twelvemonkeys_common_common-image 3.10.0
	   |-- com_twelvemonkeys_common_common-io 3.10.0
	   |-- com_twelvemonkeys_common_common-lang 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-core 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-metadata 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-webp 3.10.0
	   |-- org_quiltmc_parsers_gson 0.2.1
	   \-- org_quiltmc_parsers_json 0.2.1
```
